src/workflow/elements/gateways/OrSplitGateway.js

```diff
@@ -0,0 +1,20 @@
+const Joi = require('joi')
+const Gateway = require('./Gateway')
+
+class OrMergeGateway extends Gateway {
```

Comment:
I'm confused, the filename is orSplit but the class is OrMerge

Reply:
OrSplit doesn't make sense - goofed it up